libraries/tools/binary-compatibility-validator/reference-public-api/kotlin-stdlib-runtime-merged.txt

```diff
@@ -314,6 +314,7 @@ public final class kotlin/UByteArray : java/util/Collection, kotlin/jvm/internal
 	public static fun iterator-impl ([B)Ljava/util/Iterator;
 	public fun remove (Ljava/lang/Object;)Z
 	public fun removeAll (Ljava/util/Collection;)Z
+	public fun removeIf (Ljava/util/function/Predicate;)Z
```

Comment:
It is not expected to have changes like these in the API dump (except those directly related to the added `ExperimentalObjCName` annotation class).
I suppose they are generated when you use JDK 8 instead of JDK 6, but that will definitely cause this test to fail on a build server. So please either regenerate them having JDK 6 set up correctly, or revert them manually.